deploy/roles/ultra-edit/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+- name: Uninstall Ultra Edit
+  yum: 
+    name: "{{ item }}" #UltraEdit.x86_64
+    state: removed
+    disable_gpg_check: yes
+  with_items:
+    - "{{ ue_repo_name_version_16_18 }}"
+    - "{{ ue_repo_name_version_3_3 }}"
+    - "{{ ue_repo_name_version_16_22 }}"
+    - "{{ ue_repo_name_version_15_8}} "
+  when: custom_repo
+
+- name: Install Ultra Edit
+  yum:
+    name: "{{ ue_repo_name_version_16_22 }}" #UltraEdit.x86_64
```

Comment:
Regarding above comment, this variable would become `ue_install_version`, or something along those lines.